app/controllers/shared/api.php

```diff
@@ -139,6 +140,13 @@
             ->setQueue('v1-functions')
             ->setClass('FunctionsV1')
             ->trigger();
+        
```

Comment:
@TorstenDittmann please check if this has any effect on the REST API performance

Reply:
Couldn't measure anything, probably since the operation behind it is quite simple.